src/views/Attend/BeyondTheTalks.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <tile bgcolor="bg-white" title="Beyond The Talks" titleColor="text-wri-dark-blue">
+    <center>
+      <div class="max-w-4xl">
+        <div class="justify-center flex flex-wrap">
+          <SquareImageBlock
+            v-for="ammenity in this.amenities"
```

Comment:
Same question as above. Any reason you used `this.amenities` instead of just having `amenities`

Reply:
Same as above.  It definitely works without it as you pointed out.  Old habits die hard. It is removed now in a recent push.